docs/content/configuration/indexing-service.md

```diff
@@ -255,6 +255,7 @@ Middle managers pass their configurations down to their child peons. The middle
 |`druid.indexer.runner.classpath`|Java classpath for the peon.|System.getProperty("java.class.path")|
 |`druid.indexer.runner.javaCommand`|Command required to execute java.|java|
 |`druid.indexer.runner.javaOpts`|-X Java options to run the peon in its own JVM. Can be either a string or a json string list. Quotable parameters or parameters with spaces are encouraged to use json string lists|""|
```

Comment:
as a user: its not clear when javaOpts can be a json string list then why would I ever use javaOptsArray . 

it appears now that javaOptsArray should be able to handle all cases, can we just get rid of javaOpts and put javaOptsArray in 0.9.0 or maybe just deprecate javaOpts for now? having both of them is just weird.